CI/JenkinsfileNew

```diff
@@ -0,0 +1,59 @@
+pipeline {

```

Comment:
Can you rename this file into something more meaningful? "JenkinsfileNew" doesn't tell me what this is about.